libafl/src/events/llmp.rs

```diff
@@ -593,6 +598,52 @@ where
     pub fn to_env(&self, env_name: &str) {
         self.llmp.to_env(env_name).unwrap();
     }
+
+    /// Calling this function will tell the llmp broker that this client is exiting
+    /// This should be called from the restarter not from the actual fuzzer client
+    #[cfg(feature = "std")]
+    fn notify_death(&self, event_restarter_pid: u32, broker_port: u16) -> Result<(), Error> {
+        let mut stream = match TcpStream::connect((LLMP_CONNECT_ADDR, broker_port)) {
+            Ok(stream) => stream,
+            Err(e) => {
+                match e.kind() {
+                    ErrorKind::ConnectionRefused => {
```

Comment:
This could be an issue when exiting - the nodes may get stuck in this loop, right?
Maybe only try a few times or don't loop at all (?)